### PR TITLE
Check for successful verification

### PR DESCRIPTION
**Pull request by ianco**
Signed-off-by: Ian Costanzo <ian@anon-solutions.ca>

### CODE REVIEW

File: src/app/cred/form.component.ts [hunk 1]
```diff
@@ -98,6 +98,13 @@ export class CredFormComponent implements OnInit, OnDestroy, AfterViewInit {
       let verify_req_url = "/api/v3/credential/" + this.id + "/verify";
       let verify_proof_req = await this._httpClient.get(verify_req_url).toPromise();
 
+      // check that the proof request call was successful
+      if ("success" in verify_proof_req && !verify_proof_req["success"]) {
```
Comment (amanji) [inline src/app/cred/form.component.ts:102]: Should this be checking if the 'success' result is `true` (i.e. `!!`) or `false` (i.e. `!`)?

Comment (amanji) [reply src/app/cred/form.component.ts:102]: This looks like it will only stop loading if 'success' is `false`.

**Comment by ianco** [comment]
@amanji a bit of a hack, but updates to display "not verified" when there is an error calling the verification service, and then the error message is displayed in the console log.

**Review by amanji — APPROVED** [review]

File: src/app/cred/form.component.ts (hunk 1 above)
Comment (ianco) [reply src/app/cred/form.component.ts:102]: That's right, if success is false we stop, if success is true we continue (check for the proof request to complete)

**Pull request merged**